pytest_splunk_addon/standard_lib/event_ingestors/requirement_event_ingester.py

```diff
@@ -162,6 +162,25 @@ def get_events(self):
                                     LOGGER.info(
                                         f"sending data transport_type:forwarder/uf_file_monitor filename:{filename} "
                                     )
+                                elif transport_type in (
+                                    "scripted_input",
+                                    "scripted input",
```

Comment:
In requirement files we specify transport types- there are these two common variations in existing logs.
Transport type values are added by the dev team to requirement logs based on the ingestion type of the log in the addon

Reply:
Should we accept only 1 one of them?
Not to create options where we do not need to.

Reply:
we will document only 1 version so that we don't use others in the future, but we are allowing multiple versions to avoid going back and changing existing log files. It is the case with modinput too

Reply:
@nandinivij  Can we ensure we document official usage for input types which is sync with PSA input types?

Reply:
Also just for a record are there too many addons which use variations if not we can have them to use same version and ensure consistency and remove unwanted support from here? Do we have a list?

Reply:
@bhargav-nariyani-crest I understand the concern, but this is beyond the scope of this PR. The logs exist before we supported them to work with the requirement test. These are lexical variations and this is to facilitate requirement test work without changing existing logs. 
This does not change the transport type we are using in PSA it is the defined transport type in the requirement log.
https://github.com/splunk/pytest-splunk-addon/blob/b76294ed3fbd1dbb5a146f61deee1b86634aded6/tests/requirement_test_scripted/sample_requirement_test_scripted.log#L11

Reply:
Thanks @nandinivij .